crates/astria-sequencer/src/bridge/storage/values/address_bytes.rs

```diff
@@ -0,0 +1,61 @@
+use std::{
+    borrow::Cow,
+    fmt::{
+        self,
+        Display,
+        Formatter,
+    },
+};
+
+use astria_core::primitive::v1::ADDRESS_LEN;
+use astria_eyre::eyre::bail;
+use borsh::{
+    BorshDeserialize,
+    BorshSerialize,
+};
+use telemetry::display::base64;
+
+use super::{
+    Value,
+    ValueImpl,
+};
+use crate::accounts::AddressBytes as DomainAddressBytes;
+
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+pub(in crate::bridge) struct AddressBytes<'a>(Cow<'a, [u8; ADDRESS_LEN]>);
+
+impl<'a> Display for AddressBytes<'a> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        base64(self.0.as_slice()).fmt(f)
```

Comment:
Should assert this matches the display format of the domain type.

Reply:
As discussed offline, we can skip that since we're using Debug formatting.